DEVELOPING.md

```diff
@@ -15,6 +15,63 @@ helm install --wait \
     rancher-backup ./charts/rancher-backup
 ```
 
+### Developer Help Script
+For help with setting up the environment, check out `/scripts/deploy`. It has some helpful functions for deploying the various components used in the development environment.
+
+The function options include:
+./deploy [template/publish/minio/backup-restore/backup/remove-charts/clean/help]
+
+The following variables can be exported to access more functionality:
+
+    KUBECONFIG          Path to your cluster's kube config file.
+
+    DOCKERHUB_USER      Your docker hub username to be used for the image repo.
+                        With this exported you can publish the local image to  
+                        allow it to be to be deployed on a remote host."
+
+    USE_DOCKER_BUILDX   This flag will force the package script to use docker 
+                        buildx, setting the target platform to build for amd64 
+                        This is useful when developing on different architectures.
+
+#### Example Usage
+
+Once your development environment has been set up (i.e. have a cluster up and running), start with exporting the kubeconfig file. From there you can deploy Minio onto the cluster and then backup-restore. Once both charts have been installed you can create a backup. 
+
+```bash
+export KUBECONFIG=`pwd`/kubeconfig.yaml
+
+./deploy minio
+
+./deploy backup-restore
```

Comment:
Is this supposed to work if the user has not pushed the image at all (as described below)?

Reply:
Well the execution without the DOCKERHUB_USER tag exported would make it use the rancher docker hub image.